packages/fx-core/src/plugins/resource/function/plugin.ts

```diff
@@ -668,21 +668,22 @@ export class FunctionPluginImpl {
       "function",
       "bicep"
     );
-    const provisionTemplateFilePath = path.join(bicepTemplateDirectory, Bicep.ProvisionV2FileName);
 
-    const configTemplateFilePath = path.join(bicepTemplateDirectory, Bicep.ConfigV2FileName);
+    const provisionTemplateFilePath = path.join(bicepTemplateDirectory, Bicep.ProvisionFileName);
```

Comment:
Why change the file from v2 to v1?

Reply:
V2 in the file name is temporary. Change it to make the file name meaning clear.
The else V2 will be removed later.